packages/cli/src/commonlib/telemetry.ts

```diff
@@ -44,6 +49,10 @@ export class CliTelemetryReporter implements TelemetryReporter {
       properties = {};
     }
 
+    if (this.isFromSample !== undefined) {
+      properties[TelemetryProperty.IsFromSample] = this.isFromSample.toString();
```

Comment:
Other common properties are directly retrieved using utility functions, for example:
```
 const projectId = getProjectId(this.rootFolder);
 properties[TelemetryProperty.ProjectId] = projectId ? projectId : "";
```

But `is-from-sample` is stored in a global variable in `activate()`.

Why is `is-from-sample` different from other properties?

Reply:
This is to call core to get the property. For any command, running once is enough. If writing the logic this place, it will waste some time. Also, I think the other previous properties can be aligned with it.

Reply:
If we put "is-from-sample" property here, it will be added to all events including core and CLI levels. Please re-consider if this is aligned with vscode extension, and what's the goal of adding "is-from-sample" property.